nemoguardrails/library/domain_hallucination/files/03_scripts/e2e_eval_pipeline.py

```
@@ -0,0 +1,1030 @@
+                "error": str(exc)[:160],
+            }
+        self._dns_cache[domain] = result
+        return result
+
+    async def verify_http(self, url: str) -> Dict[str, Any]:
+        if url in self._http_cache:
+            return {**self._http_cache[url], "cache_hit": True}
+        try:
+            import httpx
+
+            async with httpx.AsyncClient(timeout=self.timeout, follow_redirects=True, verify=False) as client:
```

Ruff (0.15.15)

[error] Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)